docs/source/user_devices.rst

```diff
@@ -13,4 +13,14 @@ This import statement assumes your custom device follows the new device structur
 
 Note that both the `userlib` path and the `user_devices` folder name can be custom configured in the `labconfig.ini` file. The `user_devices` folder must be in the `userlib` path. If a different `user_devices` folder name is used, the import uses that folder name in place of `user_devices` in the above import statement.
 
-Note that we highly encourage everyone that adds support for new hardware to consider making a pull request to **labscript-devices** so that it may be added to the mainline and more easily used by other groups.
\ No newline at end of file
+Note that we highly encourage everyone that adds support for new hardware to consider making a pull request to **labscript-devices** so that it may be added to the mainline and more easily used by other groups.
+
+3rd Party Devices
+-----------------
+
+Below is a list of 3rd party devices developed by users of **labscript** that can be used via the `user_devices` mechanism described above. These repositories are not tested or maintained by the **labscript** development team. As such, there is no guarantee they will work as **labscript** is developed. They are also not guaranteed to be free of lab-specific implementation details that may prevent direct use in your apparatus. They are provided by users to benefit the community in supporting new and/or unusual devices, and can often serve as a good reference when developing your own devices. Please direct any questions regarding these repositories to their respective owners.
```

Comment:
Thanks for adding this! Apologies that these are probably overly picky...

> Below is a list of 3rd party devices developed by users of **labscript**

I'm not sure how consistent we are being but I think we generally try and refer to to **labscript-suite** rather than just **labscript**. So I would probably try and say something like "...users of the **labscript-suite**"?

> As such, there is no guarantee they will work as **labscript** is developed

"As such, there is no guarantee they will work with current or future versions of the **labscript-suite**"?

I also generally try and have each sentence on a new line so it's easier to track changes (diffs are usually formatted nicer). Single newlines shouldn't case a paragraph break I believe? I might have missed the train on imposing this styling given there are probably a lot of places already where we don't follow this! But something to consider going forward.

Reply:
I have no problem with picky while habits are being set!

I've fixed both comments, and I've gone ahead and fixed the newlines for the entire repo. Putting things on newlines is something I know to do, but often forget and get lazy so a helpful reminder is well warranted. Next time I do a doc update in any of the other repos, I'll try to remember to go in and fix all of them at once.